docs/site/sitemap.tsv

```diff
@@ -76,80 +75,82 @@ index
 			translation/translation-guide-general/references
 			translation/unique-translations
 			translation/getting-started/fixing-errors
-		index/survey-tool
+		index/survey-tool	
```

Comment:
some added white space at end of line here?

Reply:
That is deliberate; see the notes at the top. It allows github to present in tabular form.